v3/lints/cabf_smime_br/mailbox_address_from_san.go

```diff
@@ -90,12 +103,14 @@ func (l *MailboxAddressFromSAN) Execute(c *x509.Certificate) *lint.LintResult {
 	return &lint.LintResult{Status: lint.Pass}
 }
 
-func getMailboxAddressesFromDistinguishedName(name pkix.Name) []string {
+func getMailboxAddressesFromDistinguishedName(name pkix.Name, includeCN bool) []string {
 	mailboxAddresses := []string{}
 
-	for _, commonName := range name.CommonNames {
-		if util.IsMailboxAddress(commonName) {
-			mailboxAddresses = append(mailboxAddresses, commonName)
+	if includeCN {
```

Comment:
```suggestion
	if len(name.EmailAddress) == 0 {
```
This would cover more scenario's and doesn't require additional function parameters.

Reply:
would not that, probably very unlikely, produce some false positives? E.g. for a sponsor-validated certificate with a subjectDN featuring a CN which has a pseudonym with a @ character and no emailAddress RDN in subject.

Reply:
That might be possible indeed.